src/components/PortfolioTableRow/index.tsx

```diff
@@ -49,6 +49,7 @@ function PortfolioTableRow({ stake, onWithdraw }: PortfolioTableRowProps) {
                     {stake.rewards.reward && stake.rewards.reward.claimed.currency.symbol} <br />~
                     {stake.rewards.reward$ && stake.rewards.reward$.claimed
                         .add(stake.rewards.reward$.unclaimed)
+                        .multiply(100)
```

Comment:
this is a hack.
the fix should be fixing reward$.claimed and reward$.unclaimed, why are they not in the right precision?
check staking.ts